lib/runtime_options.go

```diff
@@ -58,6 +58,7 @@ type RuntimeOptions struct {
 	NoThresholds  null.Bool   `json:"noThresholds"`
 	NoSummary     null.Bool   `json:"noSummary"`
 	SummaryExport null.String `json:"summaryExport"`
+	KeyWriter     null.String `json:"keyWriter"`
```

Comment:
```suggestion
	KeyWriter     null.String `json:"-"`
```

I don't think this is used anywhere, but still, it shouldn't be configurable from any place that is JSON ... or maybe it works with `config.json`, we need to test that :thinking: 
cc @na-- WDYT, should this even be configurable from there?

Reply:
I would not encourage it so I think it should be better to not have it. But let's wait for the @na--'s opinion.